src/frontend/components/StudentLiveAdvertising/StudentLiveRegistration/index.tsx

```diff
@@ -0,0 +1,264 @@
+    try {
+      const anonymousId = uuidv4();
+      await createLiveRegistration(anonymousId, email);
+      setRegistered(true);
+    } catch (error: any) {
+      let errorMessage;
+      if (error.email && error.email[0].indexOf('already registered') > 0) {
+        errorMessage = intl.formatMessage(
+          messages.updateMailAlreadyExistingError,
+          {
+            email,
+          },
+        );
+      } else if (error.email) {
+        errorMessage = intl.formatMessage(messages.updateMailNotValidError, {
+          email,
+        });
+      } else {
+        errorMessage = intl.formatMessage(messages.updateMailDefaultError, {
+          email,
+        });
+      }
+      toast.error(errorMessage);
+    }
+  };
+
+  useLayoutEffect(() => {
+    let canceled = false;
+    const checkRegistered = async () => {
+      if (!checkLtiToken(decodedJWT)) {
+        setIsLoading(false);
+        return;
+      }
+
+      let registrations;
+      try {
+        registrations = await fetchList({
+          queryKey: ['liveregistrations'],
+          meta: {},
+        });
+      } catch (err) {
+        return;
+      }
+
+      if (canceled) {
+        return;
+      }
+      setIsLoading(false);
+      setRegistered(registrations.count > 0);
+    };
+
+    checkRegistered();
+    return () => {
+      canceled = true;
+    };
+  }, [decodedJWT]);
+
+  if (isLoading) {
+    return <Fragment />;
+  }
+
+  return (
+    <AdvertisingBox margin={{ bottom: 'large' }} pad="large">
+      <Heading level={4} color={normalizeColor('blue-active', theme)}>
+        {intl.formatMessage(messages.formTitle)}
+      </Heading>
+
+      <Box margin={{ top: 'medium', bottom: 'medium' }}>
+        {registered ? (
+          <Paragraph color={normalizeColor('blue-active', theme)}>
+            {intl.formatMessage(messages.updateSuccessfulEmail)}
+          </Paragraph>
+        ) : (
+          <Grommet theme={formTheme}>
+            <Form
+              value={values}
+              onChange={(formValues) => setValues(formValues)}
+              onSubmit={async ({ value }) => {
+                if (!value.email) {
+                  //  submit without email should not be possible since validation should fail
+                  return;
+                }
+
+                onSubmit(value.email);
+              }}
+            >
+              <FormField
+                name="email"
+                htmlFor="text-input-id"
+                contentProps={{ pad: 'none', style: { border: 'none' } }}
+                pad={false}
+                validate={{
+                  regexp:
+                    /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/,
+                  message: intl.formatMessage(
+                    messages.emailFormatValidationError,
+                  ),
+                  status: 'error',
```

Comment:
I don't know if this is needed if you use an input type email

Reply:
As discuss ealier, as far as I know we have two solutions here:
- use grommet formfield validation with a regex (however we can simplify the regex)
- use an input type email and CSS classes valid and invalid

I would rather stick to js option mainly because due to the current implementation it would requiere some selector to to be able to display the error under the field.
What do you think ?